README.md

```diff
@@ -4,14 +4,18 @@ A context menu entry for Dolphin to open VS Code at the current location.
 
 Works on the empty background or on a folder.
 
-
 ## Download
 
 <a href="https://store.kde.org/p/1413799/"><img alt="Download from the KDE Store" height="50px"
 src="https://img.shields.io/badge/KDE_Store-blue?style=for-the-badge&logo=kde"></a>
 <a href="https://github.com/Merrit/kde-dolphin-open-vscode/releases/latest"><img alt="Download from GitHub" height="50px"
 src="https://img.shields.io/badge/GitHub-blue?style=for-the-badge&logo=github"></a>
 
+## Install
+
+Copy the file `openVSCode.desktop` into the `ServiceMenus` folder:
+1. Create folders: `mkdir -p ~/.local/share/kservices5/ServiceMenus`
+2. Move file: `mv openVSCode.desktop ~/.local/share/kservices5/ServiceMenus`
```

Comment:
This is no longer the path for Plasma 6, for users on newer releases the path will have to be `~/.local/share/kio/servicemenus/`

Reply:
Thanks, I'll adjust it!